snyk-monitor/values.yaml

```diff
@@ -28,8 +28,13 @@ image:
 clusterName: ""
 
 # The snyk-monitor requires disk storage to temporarily pull container images and to scan them for vulnerabilities.
-# This value controls how much disk storage _at most_ may be allocated for the snyk-monitor. The snyk-monitor mounts an emptyDir for storage.
-temporaryStorageSize: 50Gi
+# This value controls how much disk storage _at most_ may be allocated for the snyk-monitor. The snyk-monitor mounts an emptyDir for storage. # TODO updateme
```

Comment:
There is a small `# TODO updateme` in here! 🙏

Reply:
argh

Reply:
Good spot! fixed